Libplanet/Blocks/Block.cs

```diff
@@ -19,6 +19,7 @@ public sealed class Block<T> :
         IPreEvaluationBlock<T>,
         IPreEvaluationBlock,
         IBlockHeader,
+        IActionTypeLoaderContext,
```

Comment:
How about extending `IActionTypeLoaderContext` from `IBlockHeader` instead of `Block<T>`?

Reply:
I moved the inheritance point to `IBlockMetadata`. I could bypass the compile error from the property name conflict with the `new` keyword.